apps/studio/components/interfaces/Auth/Users/UserDropdown.tsx

```diff
@@ -157,7 +158,7 @@ const UserDropdown = ({
                       }}
                     >
                       <Mail size={14} />
-                      <p>Send magic link</p>
+                      <p>{isUserConfirmed ? "Send magic link": "Send confirm sign up link"}</p>
```

Comment:
I think the email that gets sent out needs to be different as well as at the moment the same magic link email gets emailed even if the `Send confirm sign up link` is showing.

Reply:
@Hallidayo , thanks for the review, https://github.com/orgs/supabase/discussions/27425 from this issues it state that, it sends the confirmation *when* the email is not yet confirmed. Also, this is the mutation function sending the email
https://github.com/supabase/supabase/blob/master/apps/studio/components/interfaces/Auth/Users/UserDropdown.tsx#L57 and there not appear which function is sending the confirm sign up.